airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/JsonSchemaPrimitiveUtil.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.protocol.models;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.ImmutableBiMap;
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+
+public class JsonSchemaPrimitiveUtil {
+
+  public enum JsonSchemaPrimitive {
+
+    STRING,
+    NUMBER,
```

Comment:
I think this needs to declare `INTEGER` as well, since we have some tests that use `{type: integer}`. (also needs to be added to `IncrementalUtils#compareCursors`)